packages/components/src/globals/scss/_tooltip.scss

```diff
@@ -52,8 +55,26 @@
     content: '';
   }
 
+  .#{$prefix}--tooltip__content {
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
+    border: 0;
+    visibility: visible;
```

Comment:
`visibility:visible` undoes `visiblity:hidden` in an ancestor node. I'd recommend `visibility:inherit`.

Reply:
Also does this have similar purpose of `.bx--assistive-text`? https://github.com/carbon-design-system/carbon/blob/v10.3.0/packages/components/src/globals/scss/_css--helpers.scss#L17-L29